diff --git a/libusb/os/darwin_usb.c b/libusb/os/darwin_usb.c
index c7cbc38b..ed0d6a62 100644
--- a/libusb/os/darwin_usb.c
+++ b/libusb/os/darwin_usb.c
@@ -2268,7 +2268,8 @@ static int darwin_abort_transfers (struct usbi_transfer *itransfer) {
   usbi_dbg (ctx, "calling clear pipe stall to clear the data toggle bit");
 
   /* newer versions of darwin support clearing additional bits on the device's endpoint */
-  kresult = (*(cInterface->interface))->ClearPipeStallBothEnds(cInterface->interface, pipeRef);
+  // kresult = (*(cInterface->interface))->ClearPipeStallBothEnds(cInterface->interface, pipeRef);
+  kresult = 0;
 
   return darwin_to_libusb (kresult);
 }
