diff --git a/acinclude.m4 b/acinclude.m4
index df8abe148..28adc6c1f 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -249,7 +249,7 @@ AC_DEFUN([SANE_CHECK_PTHREAD],
               [Define if pthread_t is integer.])
   else
     # Until the sanei_thread implementation is fixed.
-    use_pthread=no
+    use_pthread=yes
   fi
 
   if test "$have_pthread" = "yes" ; then
diff --git a/backend/pixma/pixma_io_sanei.c b/backend/pixma/pixma_io_sanei.c
index ae780b3e7..9a3e29ad7 100644
--- a/backend/pixma/pixma_io_sanei.c
+++ b/backend/pixma/pixma_io_sanei.c
@@ -66,7 +66,7 @@
  * This is a very basic turnaround for MAC OS X
  * Button scan will not work with this wrapper */
 #ifdef __APPLE__
-# define sanei_usb_read_int sanei_usb_read_bulk
+// # define sanei_usb_read_int sanei_usb_read_bulk
 #endif
 
 
